release-0.3.md

```diff
@@ -0,0 +1,47 @@
+# Contour 0.3
+
+Heptio is pleased to announce the release of Contour 0.3.
+
+## New and improved
+
+The headline features of this release are,
+
+### TLS support
+
+Contour now supports HTTP and HTTPS ingress.
+Contour TLS support works with the [standard ingress object][1].
+You can read more about [Contour's TLS support here][2].  
+
+### More supported annotations
+
+Support for the following annotations on the `service` and `ingress` object has been added in this release.
+
+- `kubernetes.io/ingress.allow-http: "false"` will remove the ingress configration from the non TLS listener. Envoy will not serve traffic for your ingress's vhost on port 80.
+- `ingress.kubernetes.io/force-ssl-redirect: "true"` will cause Envoy to issue an unconditional 301 redirect to the HTTPS version of your site.
+
+### gRPC API is not GA
+
+Support for Envoy's version 2 gRPC based API, introduced in Contour 0.2, has been marked GA.
+The previous REST API does not support new Envoy features such as SNI.
+The REST API is now deprecated and will be removed completely in Contour 0.4.
```

Comment:
Do you mean that Contour support for the REST API is deprecated? Or that the REST API is deprecated in Envoy? If the former (as I suspect), then edit:
"Support for the REST API is deprecated and will be removed in a future version of Contour."
(Unless we really can promise a thing for a specific version? I've been taught never to mention such things ...)

Reply:
It's definitely going away in 0.4